frontend/src/App.tsx

```diff
@@ -50,25 +48,30 @@ function Users() {
 }
 
 const operations = {
-  '+': (left: number, right: number): number => left + right,
-  '-': (left: number, right: number): number => left - right,
-  '*': (left: number, right: number): number => left * right,
-  '/': (left: number, right: number): number => left / right,
-}
+  "+": (left: number, right: number): number => left + right,
+  "-": (left: number, right: number): number => left - right,
+  "*": (left: number, right: number): number => left * right,
+  "/": (left: number, right: number): number => left / right,
+};
 type CalculatorProps = {
-  left: number
-  operator: keyof typeof operations
-  right: number
-}
-function Calculator({left, operator, right}: CalculatorProps) {
-  const result = operations[operator](left, right)
+  left: number;
+  operator: keyof typeof operations;
+  right: number;
+};
+function Calculator({ left, operator, right }: CalculatorProps) {
+  fetch("http://localhost:8080/stablecoins", {
+    mode: "no-cors",
```

Comment:
this is how we'd query the server, but ideally this `http://localhost:8080` part is defined in the configs (webpack or client-side package.json)
Also `no-cors` should not be required in prod but I had to set this for the local server